src/main/java/calculator/util/StringUtils.java

```diff
@@ -0,0 +1,13 @@
+package calculator.util;
+
+/**
+ * @author : yusik
+ * @date : 2019/10/25
+ */
+public class StringUtils {
+
+    public static boolean isEmpty(String str) {
```

Comment:
메서드 이름은 `isBlank`가 적절해 보여요. 아래의 글을 참고하세요.
https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isBlank-java.lang.CharSequence-

```suggestion
    public static boolean isBlank(String str) {
```

Reply:
공백 문자일 때도 true 가 되서 그런거군요 
피드백 감사합니다! 😃